Outbox not found

```
SUMMARY]
javax.mail.FolderNotFoundException: Outbox not found
com.flowcrypt.email.util.exception.ManualHandledException: Handled manually:


[STACK_TRACE]
com.flowcrypt.email.util.exception.ManualHandledException: Handled manually:
        at com.flowcrypt.email.util.exception.ExceptionUtil$Companion.handleError(ExceptionUtil.kt:30)
        at com.flowcrypt.email.jetpack.workmanager.ForwardedAttachmentsDownloaderWorker$downloadForwardedAtts$2.invokeSuspend(ForwardedAttachmentsDownloaderWorker.kt:173)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
        at kotlinx.coroutines.internal.LimitedDispatcher.run(LimitedDispatcher.kt:42)
        at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:95)
        at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:570)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:749)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:677)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:664)
Caused by: javax.mail.FolderNotFoundException: Outbox not found
        at com.sun.mail.imap.IMAPFolder.checkExists(IMAPFolder.java:431)
        at com.sun.mail.imap.IMAPFolder.open(IMAPFolder.java:1044)
        at com.sun.mail.imap.IMAPFolder.open(IMAPFolder.java:977)
        at com.flowcrypt.email.jetpack.workmanager.ForwardedAttachmentsDownloaderWorker$getNewMsgState$2.invokeSuspend(ForwardedAttachmentsDownloaderWorker.kt:212)
        at com.flowcrypt.email.jetpack.workmanager.ForwardedAttachmentsDownloaderWorker$getNewMsgState$2.invoke(Unknown Source:8)
        at com.flowcrypt.email.jetpack.workmanager.ForwardedAttachmentsDownloaderWorker$getNewMsgState$2.invoke(Unknown Source:4)
        at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:89)
        at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:166)
        at kotlinx.coroutines.BuildersKt.withContext(Unknown Source:1)
        at com.flowcrypt.email.jetpack.workmanager.ForwardedAttachmentsDownloaderWorker.getNewMsgState(ForwardedAttachmentsDownloaderWorker.kt:186)
        at com.flowcrypt.email.jetpack.workmanager.ForwardedAttachmentsDownloaderWorker.access$getNewMsgState(ForwardedAttachmentsDownloaderWorker.kt:54)
        at com.flowcrypt.email.jetpack.workmanager.ForwardedAttachmentsDownloaderWorker$downloadForwardedAtts$2.invokeSuspend(ForwardedAttachmentsDownloaderWorker.kt:144)
        ... 8 more
        Suppressed: java.lang.IllegalStateException: This operation is not allowed on a closed folder
                at com.sun.mail.imap.IMAPFolder.close(IMAPFolder.java:1486)
                at com.sun.mail.imap.IMAPFolder.close(IMAPFolder.java:1460)
                at javax.mail.Folder.close(Folder.java:645)
                at kotlin.jdk7.AutoCloseableKt.closeFinally(AutoCloseable.kt:64)
                at com.flowcrypt.email.jetpack.workmanager.ForwardedAttachmentsDownloaderWorker$getNewMsgState$2.invokeSuspend(ForwardedAttachmentsDownloaderWorker.kt:211)
                ... 16 more
```

--- a/FlowCrypt/src/androidTest/assets/messages/info/standard_msg_info_plaintext_outbox.json
+++ b/FlowCrypt/src/androidTest/assets/messages/info/standard_msg_info_plaintext_outbox.json
@@ -1,0 +1,55 @@
+{
+  "encryptionType": "STANDARD",
+  "inlineSubject": "Standard message - plaintext",
+  "msgBlocks": [
+    {
+      "complete": true,
+      "content": "\n  \u003c!DOCTYPE html\u003e\u003chtml\u003e\n    \u003chead\u003e\n      \u003cmeta name\u003d\"viewport\" content\u003d\"width\u003ddevice-width\" /\u003e\n      \u003cstyle\u003e\n        body { word-wrap: break-word; word-break: break-word; hyphens: auto; margin-left: 0px; padding-left: 0px; }\n        body img { display: inline !important; height: auto !important; max-width: 95% !important; }\n        body pre { white-space: pre-wrap !important; }\n        body \u003e div.MsgBlock \u003e table { zoom: 75% } /* table layouts tend to overflow - eg emails from fb */\n      \u003c/style\u003e\n    \u003c/head\u003e\n    \u003cbody\u003e\u003cdiv class\u003d\"MsgBlock plain\" style\u003d\"background: white;padding-left: 8px;min-height: 50px;padding-top: 4px;padding-bottom: 4px;width: 100%;border: none;\"\u003e\n\n    \n  \n  \n    \u003cpre\u003eIt\u0027s a standard message with plaintext\u003c/pre\u003e\u003c/div\u003e\u003c!-- next MsgBlock --\u003e\n\u003c/body\u003e\n  \u003c/html\u003e",
+      "type": "plainHtml"
+    }
+  ],
+  "msgEntity": {
+    "cc": [],
+    "email": "default@flowcrypt.test",
+    "flags": "\\SEEN",
+    "folder": "Outbox",
+    "from": [
+      {
+        "address": "denbond7@flowcrypt.test",
+        "encodedPersonal": "Denis Bondarenko",
+        "personal": "Denis Bondarenko"
+      }
+    ],
+    "fromAddress": "Denis Bondarenko \u003cdenbond7@flowcrypt.test\u003e",
+    "hasAttachments": false,
+    "id": 50,
+    "isSeen": true,
+    "msgState": "NONE",
+    "replyTo": "Denis Bondarenko \u003cdenbond7@flowcrypt.test\u003e",
+    "replyToAddress": [
+      {
+        "address": "denbond7@flowcrypt.test",
+        "encodedPersonal": "Denis Bondarenko"
+      }
+    ],
+    "sentDate": 1601886754000,
+    "subject": "Standard message - plaintext",
+    "to": [
+      {
+        "address": "default@flowcrypt.test"
+      }
+    ],
+    "toAddress": "default@flowcrypt.test",
+    "uid": 5
+  },
+  "verificationResult": {
+    "isEncrypted": false,
+    "isSigned": false,
+    "hasMixedSignatures": false,
+    "isPartialSigned": false,
+    "hasUnverifiedSignatures": false,
+    "hasBadSignatures": false
+  },
+  "origMsgHeaders": "Return-Path: \u003cdenbond7@flowcrypt.test\u003e\r\nDelivered-To: default@flowcrypt.test\r\nReceived: from mail.flowcrypt.test\r\n\tby mail.flowcrypt.test with LMTP\r\n\tid WF4PDSLael+dCQAAZlazJA\r\n\t(envelope-from \u003cdenbond7@flowcrypt.test\u003e)\r\n\tfor \u003cdefault@flowcrypt.test\u003e; Mon, 05 Oct 2020 08:32:34 +0000\r\nReceived: from localhost (localhost [127.0.0.1])\r\n\tby mail.flowcrypt.test (Postfix) with ESMTP id 347E7A20940\r\n\tfor \u003cdefault@flowcrypt.test\u003e; Mon,  5 Oct 2020 08:32:34 +0000 (UTC)\r\nTo: default@flowcrypt.test\r\nFrom: Denis Bondarenko \u003cdenbond7@flowcrypt.test\u003e\r\nX-Pep-Version: 2.0\r\nMessage-ID: \u003c8fcbd0f1-e33b-9b5e-51ee-79d7c639f0bc@flowcrypt.test\u003e\r\nDate: Mon, 5 Oct 2020 11:32:34 +0300\r\nMime-Version: 1.0\r\nSubject: Standard message - plaintext\r\nContent-Type: multipart/alternative;\r\n boundary\u003d\"------------D3F69D95534F587EC1BE35A3\"\r\nContent-Language: en-US",
+  "text": "It\u0027s a standard message with plaintext"
+}

--- a/FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/MessageDetailsFlowTest.kt
+++ b/FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/MessageDetailsFlowTest.kt
@@ -823,4 +823,24 @@ class MessageDetailsFlowTest : BaseMessageDetailsFlowTest() {
       )
     )
   }
+
+  @Test
+  fun testViewsVisibilityOfOutgoingMessages() {
+    val incomingMsgInfo = requireNotNull(
+      getMsgInfo(
+        "messages/info/standard_msg_info_plaintext_outbox.json",
+        "messages/mime/standard_msg_info_plaintext.txt"
+      )
+    )
+
+    val msgEntity = incomingMsgInfo.msgEntity
+    launchActivity(msgEntity)
+
+    onView(withId(R.id.imageButtonReplyAll))
+      .check(matches(not(isDisplayed())))
+    onView(withId(R.id.imageButtonMoreOptions))
+      .check(matches(not(isDisplayed())))
+    onView(withId(R.id.layoutReplyButtons))
+      .check(matches(not(isDisplayed())))
+  }
 }

--- a/FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/fragment/MessageDetailsFragment.kt
+++ b/FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/fragment/MessageDetailsFragment.kt
@@ -465,8 +465,8 @@ class MessageDetailsFragment : BaseFragment<FragmentMessageDetailsBinding>(), Pr
   private fun showIncomingMsgInfo(msgInfo: IncomingMessageInfo) {
     this.msgInfo = msgInfo
     this.msgEncryptType = msgInfo.encryptionType
-    binding?.imageButtonReplyAll?.visibility = View.VISIBLE
-    binding?.imageButtonMoreOptions?.visibility = View.VISIBLE
+    binding?.imageButtonReplyAll?.visibleOrGone(!args.messageEntity.isOutboxMsg())
+    binding?.imageButtonMoreOptions?.visibleOrGone(!args.messageEntity.isOutboxMsg())
     isAdditionalActionEnabled = true
     activity?.invalidateOptionsMenu()
     msgInfo.localFolder = args.localFolder
@@ -1024,7 +1024,7 @@ class MessageDetailsFragment : BaseFragment<FragmentMessageDetailsBinding>(), Pr
       binding?.layoutReplyButtons?.layoutReplyButton?.setOnClickListener(this)
       binding?.layoutReplyButtons?.layoutFwdButton?.setOnClickListener(this)
       binding?.layoutReplyButtons?.layoutReplyAllButton?.setOnClickListener(this)
-      binding?.layoutReplyButtons?.root?.visibility = View.VISIBLE
+      binding?.layoutReplyButtons?.root?.visibleOrGone(!args.messageEntity.isOutboxMsg())
     }
   }
 

